seeker/message-center.html

```diff
@@ -0,0 +1,138 @@
+
+    <!-- Custom Styles -->
+    <link rel="stylesheet" href="../css/base.css">
+    <link rel="stylesheet" href="../css/global.css">
+    <link rel="stylesheet" href="../css/message-center.css">
+</head>
+
+<body>
+    <div class="wrapper">
+      <header>
+        <a href="/" class="logo">
+          <img src="/img/logo_ref.png">
+          <span>PetPal</span>
+        </a>
+        <nav class="header__nav">
+          <div class="nav__menu">
+            <label for="nav__menu-toggle"><div class="nav__hamburger"></div></label>
+            <input type="checkbox" name="nav__menu-toggle" id="nav__menu-toggle" required>          
+            <ul>
+              <li><a href="#" class="nav__link">About Us</a></li>
+              <li><a href="login.html" class="nav__link">Adopt</a></li>
+              <li><a href="login.html" class="nav__link">Shelters</a></li>
+              <li><a href="#" class="nav__link">Donate</a></li>
+            </ul>
+          </div>
+          <div class="nav__icons">
+            <i class="material-icons">search</i>
+            <i class="material-icons">notifications</i>
+            <i class="material-icons">account_circle</i>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <div class="main__wrapper">
+
+            <p class="page-title">Message Center</p>
+            <div class="msg-container">
+                <div class="msg-nav">
+                    <div id="inbox" class="msg-nav-item">Inbox</div>
+                    <div id="sent" class="msg-nav-item">Sent</div>
+                </div>
+                <div class="msg-row">
+                    <div class="msg-info">
+                        <div class="msg-subject">Subject</div>
+                        <div class="msg-from">from: shelter1</div>
+                        <div class="msg-time">yy/mm/dd hh:mm</div>
+                    </div>
+
+                    <div class="btn-container">
+                        <a class="btn" href="">Reply</a>
+                    </div>
+                </div>
+
+                <div class="msg-row">
+                    <div class="msg-info">
+                        <div class="msg-subject">Subject</div>
+                        <div class="msg-from">from: shelter1</div>
+                        <div class="msg-time">yy/mm/dd hh:mm</div>
+                    </div>
+
+                    <div class="btn-container">
+                        <a class="btn" href="">Reply</a>
+                    </div>
+                </div>
+
+
+                <div class="msg-row">
+                    <div class="msg-info">
+                        <div class="msg-subject">Subject</div>
+                        <div class="msg-from">from: shelter1</div>
+                        <div class="msg-time">yy/mm/dd hh:mm</div>
+                    </div>
+
+                    <div class="btn-container">
+                        <a class="btn" href="">Reply</a>
```

Comment:
Should these reply buttons link to `message-new.html`? Or is there some other page design in mind?